frontend/src/components/SideBar.tsx

```diff
@@ -35,17 +38,17 @@ export default function SideBar() {
     React.useMemo(() => {
       return [
         {type: 'link', text: t('global.menu.home'), href: '/home'},
-        isUser()
+        isAdmin()
```

Comment:
nitpick: I guess we can remove entirely these ternaries since all users are administrators, even in other places of the portal

Reply:
Absolutely. Working on it